File: Feitos em C Sharp/2551.cs
using System;
namespace ConsoleApp26 {
    class Program {
        static void Main(string[] args) {
			while (true) {
				string auxTimes = Console.ReadLine();
				if (string.IsNullOrEmpty(auxTimes)) {
					break;
				}
				int times = int.Parse(auxTimes);
				double[] vet = new double[times]; 
				double aux;
				double record = 0.0;
				for (int i = 0; i < times; i++) {
					string[] input = Console.ReadLine().Split();
					vet[i] = double.Parse(input[0]);
					aux = double.Parse(input[1]);
					double sub = aux / vet[i];
					if (i == 0) {
						record = sub;
						Console.WriteLine(i+1);
					}
					else if (sub > record) {
						record = sub;
						Console.WriteLine(i+1);
					}
				}
			}
		}
    }
}

File: Feitos em Python 3/2551.py
while True:
    try:
        times=int(input())
        i=0
        record=0
        while i<times:
            ent=input().split()
            t=float(ent[0])
            d=float(ent[1])
            sub = d/t
            if i==0:
                record = sub
                print(i+1)
            elif sub>record:
                record = sub
                print(i+1)
            i+=1
    except EOFError:
        break